content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -17,6 +17,35 @@ changelogItems:
 - [From code search toward a code intelligence platform](/blog/code-search-to-code-intelligence) - the longer-term vision/story
 - [Secure, easy, and scalable Sourcegraph Cloud managed instances for the enterprise](/blog/enterprise-cloud) - the Cloud announcement (including SOC2 Type 2, etc.) that coincides with 4.0
 
+## TL;DR
+
+`TODO(sqs): The section headers with emojis are what I think we should use, but the wording of the bullet points within are WIP`
+
+##### 🌅 Code intelligence: more useful, more times daily for every dev
```

Comment:
@sqs @iskyOS - I really like the framing of finding/getting answers and information about your code vs simply finding things in your code. This comes through in both of your docs ([Alex](https://docs.google.com/document/d/15BDzRycLoy9X9zw-Yf3l1DY5O50pbO-rmh7AYzLjL5I/edit#), [SQS](https://docs.google.com/document/d/1EAXfMBidVX1WJERdtnU8x-urrjpnc-80-PoOzeQPNpw/edit)), but isn't apparent in this headline. I think things like precise code nav + auto-indexing, simple UI, smarter code search, and high-level aggregations all support answering questions/learning about your code. 

Here are a few alts:
- Answer questions about your codebase with code intelligence: more useful, more times daily for every dev
- Understand your code with code with code intelligence: more useful, more times daily for every dev
- Get answers with code intelligence: more useful, more times daily for every dev
- Understand with code intelligence: more useful, more times daily for every dev

Reply:
@abschu I agree, I think this headline could do a better job of summing up the value of the section (and in this case, code intelligence won't be well defined by our audience yet)

I think I like this one most: 
"Get answers with code intelligence: more useful, more times daily for every dev"